custom_components/matter_experimental/entity.py

```diff
@@ -59,20 +58,20 @@ async def init_matter_device(self) -> None:
 
         try:
             # Subscribe to updates.
-            async with async_timeout.timeout(5):
```

Comment:
Without a timeout startup can hang forever if the node never responds?

Reply:
Matter SDK will timeout at one point... Maybe we should add a (higher) timeout at one point, but not sure what time that should be